charts/stable/tsn-ranksystem/values.yaml

```diff
@@ -72,6 +76,30 @@ workload:
                   else
                     echo "Install script doesnt exists, no further action is needed."
                   fi
+  ranksystemcron:
+    enabled: true
+    type: CronJob
```

Comment:
You can set the condition here instead of having all this in common yaml 

eg enabled: “{{ .val.cron.enabledblanla. }}”

Reply:
OK. Had that before but copied how qbittorrent does it. 😅
Is the cronjob fine in general?

Reply:
i thought we love the common.yaml way, next patch update will have it.

Reply:
The less code the better 😉
Common.yaml is used on complex
Conditions usually